packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -728,6 +728,13 @@ export class SDKClient {
         throw e;
       }
 
+      // Log the target node account ID, right now, it's populated only for MaxAttemptsOrTimeout error
+      if (e?.nodeAccountId) {
+        this.logger.info(
+          `${requestDetails.formattedRequestId} Transaction failed to execute against node with id: ${e.nodeAccountId}`
+        );
+      }
+
```

Comment:
This is in executeTransaction, should we do it for executeAllTransaction as well? 

Or actually maybe in a more general place like in the EthImpl.sendRawTransactionErrorHandler()?

Reply:
Added in the `executeAllTransaction`, good catch!

`sendRawTransactionErrorHandler` uses the `SDKClientError` wrapper which doesn't have the plain SDK error object. I'm thinking about adding it but I decided it's not worth it adding more complexity for just one log.

Reply:
Ah okay that makes sense! Okay executeTransaction and executeAllTransaction should be enough then as most of the methods in SDKClient class would eventually reroute to those two execution methods! Thanks for the explanation!